### PR TITLE
feat added: product listing page

### PR DESCRIPTION
**Pull request by suryxks**
added

- Product listings
- Filters
Live preview[Here](https://deploy-preview-2--nectar-decor.netlify.app/pages/products.html)

### CODE REVIEW

File: pages/products.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+.filters {
+    display: flex;
+    flex-direction: column;
+    padding: 1rem;
+}
+
+.filter {
+    accent-color: var(--cta);
```
Comment (someshtiwarii) [inline pages/products.css:16]: Color variable naming can be different if you wish, like cta-color

File: pages/products.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    margin: 0;
+}
+
+.vertical-card {
+    display: flex;
+    flex-direction: column;
+    max-width: 14rem;
+    border: 2px var(--border);
+    box-shadow:
+        2px 4px 8px hsl(0deg 0% 0% / 0.25);
```
Comment (himadri2110) [inline pages/products.css]: Suggestion: You can use a CSS variable for `box-shadow` as you're using it at multiple places.

**Review by someshtiwarii — APPROVED** [review]
Clean code! Looks good to merge !

**Review by himadri2110 — APPROVED** [review]
Good to go!

**Review by nshanbhag20001412 — APPROVED** [review]
Looks good  to go. You can updates the links for cart and wishlist in the main page

**Review by palash23jain — APPROVED** [review]
good use of CSS variables :) good to go

File: pages/products.css (hunk 2 above)
Comment (suryxks) [reply pages/products.css]: Thanks for pointing it out i have made the changes and will merge the pr

**Pull request merged**